iati/core/resources/test_data/202/invalid_dateorder.xml

```diff
@@ -1,12 +1,42 @@
 <?xml version="1.0"?>
-
+<!--dates in incorrect order-->
 <root_element>
-  <element1 type="1">
-   <start iso-date="2017-11-01" /><!-- Dates are not in the correct order -->
-   <end iso-date="2017-03-01" />
- </element1>
-  <element2 type="1">
-   <start iso-date="01-12-2007"/><!-- Dates are not in the correct xsd:date format -->
-   <end iso-date="02-11-2005"/>
- </element2>
+  <element1>2017-08-16</element1><!--elements in wrong chronological order-->
+  <element2>2016-08-16</element2>
+  <element3>2017-08-16</element3><!--elements have identical values-->
+  <element4>2017-08-16</element4>
+  <element5>2016-08-16</element5><!--`more` as chronologically before NOW-->
+  <element6>9999-12-01</element6><!--`less` as chronologically after NOW-->
+  <nest>
+    <element7>2017-08-16</element7><!--elements in wrong chronological order when elements are nested-->
+    <element8>2016-08-16</element8>
+  </nest>
+  <element9>2017-08-16</element9><!--element is used by `less` and `more`-->
+  <element10>2017-08-16</element10><!--multiple `less` elements that are identical but chronologically after `more`-->
```

Comment:
Related tests that don't seem to be here:

* Multiple `more|less` elements that are different but chronologically correct
* Multiple `more|less` elements that are different but chronologically incorrect
* Multiple `more|less` elements that are different but chronologically mixed

Reply:
Already done in test data for dateorder format. Will add comment explaining this.

Reply:
done